README.md

```diff
@@ -63,6 +63,26 @@ The following attributes are available for all models of `viamrtsp` cameras:
 > The above is a raw JSON configuration for an `rtsp` model.
 > To use another provided model, change the "model" string.
 
+## RTSP Stream Discovery
+### Using `DiscoverComponents`
+In addition to being able to manually specify RTSP addresses to stream from, `viamrtsp` also offers a discovery utility to search for IP cameras connected to your LAN
+and return their respective RTSP addresses.
+
+Each model in `viamrtsp` registers a `Discover` method that can be invoked via a `DiscoverComponents` robot API call. See [examples/discovery-client](examples/discovery-client/client.go) for an example usage of discovery.
+
+As of time of writing, Viam supports `DiscoverComponents` in Flutter and Golang.
```

Comment:
```suggestion
As of time of writing, Viam supports `DiscoverComponents` in Flutter, Python, Typescript and Golang.
```
unless your specific implementation is only supported in flutter and golang

this is helpful bc in docs we say it's supported in Python, Go, and TypeScript, but I do see it in Flutter-- I think we just haven't automated pulling in flutter sdk docs from the robot api yet, I'll make a ticket to do that

Reply:
It is indeed supported only in Flutter and Golang atm

Reply:
sorry, just want to clarify, do you mean for all components/overall?

Reply:
I might be totally wrong. I see it in Python https://github.com/search?q=repo:viamrobotics/viam-python-sdk%20discover&type=code

However, I don't know if anyone's tested it in a long time

cc: @randhid

Reply:
Dont we fire discover requests from Typescript SDK for fancy webcam builder?

Reply:
Good point. Let's include all SDKs then